virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc

```diff
@@ -9,6 +9,11 @@ You can import a single Red Hat Virtualization (RHV) virtual machine into your {
 
 include::modules/virt-importing-vm-prerequisites.adoc[leveloffset=+1]
 
+The following table describes the local and shared persistent storage that support VM import.
```

Comment:
This line probably makes more sense in the module itself as an introduction to the table. you can then drop it from the two assemblies.

Reply:
It's not my module and it's used in other assemblies.

Reply:
Conditionals are used extensively in this module to separate content; not sure how this is any different

Reply:
Correct. I'll look at the module again and see whether it makes sense to move the line.